lib/decompress/zstd_decompress_block.c

```diff
@@ -789,7 +789,7 @@ ZSTD_initFseState(ZSTD_fseState* DStatePtr, BIT_DStream_t* bitD, const ZSTD_seqS
     DStatePtr->state = BIT_readBits(bitD, DTableH->tableLog);
     DEBUGLOG(6, "ZSTD_initFseState : val=%u using %u bits",
                 (U32)DStatePtr->state, DTableH->tableLog);
-    BIT_reloadDStream(bitD);
+    BIT_reloadUnfinishedDStream(bitD);
```

Comment:
I’d be surprised if this impacts performance since it is only called once per block